src/leap/mail/imap/mailbox.py

```diff
@@ -639,21 +533,41 @@ def _do_fetch(self, messages_asked, uid, d):
 
         sequence = False
         # sequence = True if uid == 0 else False
```

Comment:
It's confusing to have two different things called sequence. One is this boolean, that I'm not sure what it means, and the other the sequence of filtered messages. Can we rename this boolean to something more meaningful, or remove it as it don't do anything.